### ISSUE TITLE
 error: expected class-name before '(' token

### ISSUE THREAD

**Issue by Zarathustra101** [CopernicaMarketingSoftware/PHP-CPP#487]
Got error message, when using library:

```
error: expected class-name before '(' token
   43 |     Throwable(const std::string &message) : std::runtime_error(message) {}
```

Solution:

add 
`#include <stdexcept>`
in
`include/throwable.h`

**Comment by edhelas** [comment]
So  #440 fixes it

**Comment by carlos-granados** [comment]
There is a maintained fork where a PR to fix this issue has been merged, you can find it here: https://github.com/fast-debug/PHP-CPP

**Comment by robocoder** [comment]
Fixed in 2.4.0 (c2b36b9843b4b98f67342f6c0a956d1217bcb948)
